#!/usr/bin/env python
PACKAGE = "irob_vision_support"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("bg_l_h",    int_t,    0, "Background lower hue", 90,  0, 255)
gen.add("bg_l_s",    int_t,    0, "Background lower saturation", 0,  0, 255)
gen.add("bg_l_v",    int_t,    0, "Background lower value", 0,  0, 255)
gen.add("bg_u_h",    int_t,    0, "Background upper hue", 180,  0, 255)
gen.add("bg_u_s",    int_t,    0, "Background upper saturation", 255,  0, 255)
gen.add("bg_u_v",    int_t,    0, "Background upper value", 255,  0, 255)


exit(gen.generate(PACKAGE, "irob_vision_support", "Fiducials"))
